widgets/search-box/__tests__/search-box-test.js

```diff
@@ -124,7 +170,7 @@ describe('search-box()', () => {
     var input;
     beforeEach(() => {
       container = document.createElement('div');
-      input = document.createElement('input');
+      input = createHTMLNodeFromString('<input />');
```

Comment:
Hmm, this was a refactoring leftover, I did not really need to change this test. The `createHTMLNodeFromString` is needed if we want to specify some attributes to the element (and not only its tagname).

And yes, we have https://github.com/algolia/instantsearch.js/blob/feat/wrap-searchbox-352/widgets/search-box/__tests__/search-box-test.js#L80-L93 to test that an `input` can be targeted.